### PR TITLE
[BULK] DocuTune - Rebranding of Azure Cosmos DB APIs

### PR DESCRIPTION
**Pull request by alexbuckgit**
DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review and merge within five business days or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 1c5ab0c9-80d8-4820-b550-6b2ddb22d568
InitiativeId: docutune-cosmos-db-rebrand-2022-10
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: docs/java/java-webapp.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ Open the **Output** window in VS Code to view the deployment logs. Once the depl
 
 [Azure Cosmos DB](https://learn.microsoft.com/azure/cosmos-db/introduction) is a globally distributed database service that allows developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
 
-The [Spring Boot Starter](https://learn.microsoft.com/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-cosmos-db) makes it easy to store data in and retrieve data from your Azure Cosmos DB with SQL API.
+The [Spring Boot Starter](https://learn.microsoft.com/azure/developer/java/spring-framework/configure-spring-boot-starter-java-app-with-cosmos-db) makes it easy to store data in and retrieve data from your Azure Cosmos DB for NoSQL database.
```
Comment (gregvanl) [inline docs/java/java-webapp.md:142]: I don't see NoSQL mentioned in the linked to article?

**Comment by gregvanl** [comment]
Hi @alexbuckgit, any update on this PR? I had one comment

**Comment by gregvanl** [comment]
Labeling with `don't merge` since we have a 14 day SLA on PRs

**Comment by gregvanl** [comment]
Dependent on https://github.com/MicrosoftDocs/azure-dev-docs-pr/pull/3769

File: docs/java/java-webapp.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/java/java-webapp.md:142]: Linked article has been updated.

**Comment by alexbuckgit** [comment]
@gregvanl - PR is ready for merge, thanks!

#sign-off

**Pull request merged**